content/joysticks/winwing/winwing-cdu/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: WINWING CDU
+weight: 50
+---
+
+WINWING CDU display support is automatic on run, if a supported plane and a CDU are detected. To prevent MobiFlight from taking over the CDU, disable it in `Extras->Settings->Peripherals`.
```

Comment:
```suggestion
WINWING CDU display support is automatic on run, if a supported plane and a CDU are detected. To prevent MobiFlight from taking over the CDU, disable it by following the [disabling joysticks](/joysticks/disabling/) guide.
```